### DISCUSSION TITLE
Get model structure from trial folder

### DISCUSSION THREAD

**Discussion by Charlespoole** [keras-team/autokeras#1627] [Q&A]
I have already get satisfying results from certain trial, but since interrupting the auto_training processing won't yield the best model, I was wondering how can I get the model structure from the trial folder, and refining the model from there by keras?  Attached is the zipped trial folder. [trial_cae63f28c56c02c8d86d9f355e09fb6e.zip](https://github.com/keras-team/autokeras/files/7133241/trial_cae63f28c56c02c8d86d9f355e09fb6e.zip)

**Comment by haifeng-jin** [comment]
How do you tell it is not yielding the best model?
